mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/SerializableQuotaValue.java

```diff
@@ -43,6 +46,27 @@ public SerializableQuotaValue(T value) {
         this.value = value;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (o instanceof SerializableQuotaValue<?>) {
+            @SuppressWarnings("ConstantConditions") SerializableQuotaValue<?> that = (SerializableQuotaValue<?>) o;
```

Comment:
what's that? It adds a warning (unknown warning) in my eclipse

Reply:
It looks like it's not needed in the end